### ISSUE TITLE
[1.3-modules-beta.2] Importing twilio fails because underscore isn't imported correctly

### ISSUE THREAD

**Issue by nathan-muir** [meteor/meteor#5909]
Installing `twilio` under `node_modules` doesn't work - where-as `Npm.require("twilio")` works within a package.

Example app at `https://github.com/nathan-muir/meteor-1.3-npm-issue` just `npm install && meteor`.

Error thrown:

```
TypeError: Object [object Object] has no method 'extend'
    at meteorInstall.app.node_modules.twilio.lib.AccessToken.js (app/node_modules/twilio/lib/AccessToken.js:21:1)
    at fileEvaluate (packages/modules/.npm/package/node_modules/install/install.js:183:1)
    at require (packages/modules/.npm/package/node_modules/install/install.js:75:1)
    at meteorInstall.app.node_modules.twilio.lib.index.js (app/node_modules/twilio/lib/index.js:31:1)
    at fileEvaluate (packages/modules/.npm/package/node_modules/install/install.js:183:1)
    at require (packages/modules/.npm/package/node_modules/install/install.js:75:1)
    at meteorInstall.app.server.test.js (/Users/nathan/Projects/meteor-npm-issue/.meteor/local/build/programs/server/app/app.js:41:15)
    at fileEvaluate (packages/modules/.npm/package/node_modules/install/install.js:183:1)
    at require (packages/modules/.npm/package/node_modules/install/install.js:75:1)
    at /Users/nathan/Projects/meteor-npm-issue/.meteor/local/build/programs/server/app/app.js:39549:1
```

Peeking at `AccessToken.js` shows the following:

```
var _ = require('underscore'); // 1
/* snip */
_.extend(IpMessagingGrant.prototype, { // 21
  key: 'ip_messaging',
```

So, it appears the issue is `require('underscore')` isn't importing the right version / one with the correct exports.

**Comment by nathan-muir** [comment]
Looks like it was importing the Meteor Package named `underscore` and not the npm-installed `underscore` from twilio's dependencies.

I've created a patch at #5915 which uses NPM scopes to separate Meteor Package names from NPM module names.

**Comment by avital** [comment]
Fixed by #5933, will be out with `1.3-modules-beta.3` (currently unreleased)
